sdk/lib/opentelemetry/sdk/trace/samplers/probability_sampler.rb

```diff
@@ -26,51 +30,51 @@ def initialize(probability, hints:, ignore_parent:, apply_to_root_spans:, apply_
           #
           # Callable interface for probability sampler. See {Samplers}.
           def call(trace_id:, span_id:, parent_context:, hint:, links:, name:, kind:, attributes:)
-            take_hint(hint) ||
-              use_parent_sampling(parent_context) ||
-              use_link_sampling(links) ||
-              maybe_dont_apply(parent_context) ||
-              use_probability_sampling(trace_id) ||
-              NOT_RECORD
-          end
+            ignore(links, name, kind, attributes)
```

Comment:
I'm not positive that this makes things any more clear (I had to look the the implementation of ignore to know for sure what was happening). I think the comment would be a less surprising way to indicate the ignored params.